api/openapi-spec/swagger.json

```diff
@@ -6559,6 +6559,42 @@
       ],
       "type": "object"
     },
+    "io.k8s.api.core.v1.EphemeralContainers": {
```

Comment:
this line is sort of suspicious. why are the core/v1 types added upon enabling alpha features?

btw openapi spec is maintained by api-machinery i believe.

Reply:
this alpha feature is a subresource of v1/pods... subresources don't have independent versions, so they inherit the REST API version of their parent object

Reply:
As we have more more and more subresources that are now additions/modifications to stable APIs, yet themselves are not stable, it becomes harder to understand what should be part of Conformance.

Is there something within the openapi generated that could add some metadata regarding what flag it is behind, and if that flag is `off/on-by-default`, or `forced-off/on`?

Reply:
Yeah, while @liggitt is stating today's truth, it seems pretty broken to me.

I know we are not adding sub-resources every day, but I think ti just speaks to the larger issue of non-GA things (fields, now subresources) on GA paths.